src/sentry/templates/sentry/auth_close.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <script>
+  // sends message to window that opened this popup if user is authenticated and closes the window
+  function extractOrigin() {
+    var pathArray = location.href.split( '/' );
+    var protocol = pathArray[0];
+    var host = pathArray[2];
+    var origin = protocol + '//' + host;
+    return origin;
+  }
+
+    (function checkAuthChange() {
+      var loggedIn = '{{ logged_in }}';
+      if (window.opener && loggedIn === 'True') {
+        window.opener.postMessage('User is logged in', extractOrigin());
```

Comment:
It's a little weird for me to pass a symbolic value as a sentence/phrase. Can we change to something like:

```javascript
window.opener.postMessage('user_authenticated', extractOrigin()); // or user_logged_in
```